models/reviews.models.js

```diff
@@ -40,8 +40,42 @@ exports.fetchCommentsByReviewId = (review_id) => {
     })
 };
 
-exports.fetchAllReviews = () => {
-    return db.query(`SELECT reviews.owner, reviews.title, reviews.review_id, reviews.category, reviews.review_img_url, reviews.created_at, reviews.votes, COUNT(comments.review_id) AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY created_at DESC`)
+exports.fetchAllReviews = (category = "*", sort_by = "created_at", order = "DESC") => {
+
+    const validCategories = ['euro_game', 'social_deduction', 'dexterity', "children's_games", "*"];
```

Comment:
This way of checking the category might work for now but it's very brittle. 

What if there are more categories in the future? How would your model deal with that?

Also there might be some difference between the test data and the dev data?

Do we need to check the category, what would happen if we didn't?

Reply:
Also what's your reasoning for accepting '*' as a valid category?

Reply:
> Also what's your reasoning for accepting '*' as a valid category?

That for if no category was passed. It would default to */all when entered into the query.